pypesto/sampling/theano.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+
+from ..objective import Objective
+from ..problem import Problem
+
+try:
+    import theano.tensor as tt
+    from theano.gof.null_type import NullType
+except ImportError:
+    tt = NullType = None
+
+
+class TheanoLogProbability(tt.Op):
+    """
+    Theano wrapper around a (non-normalized) log-probability function.
+
+    Parameters
```

Comment:
Should be "Attributes", not? Doesn't sphinx care?

Reply:
I think we use parameters in many parts ... sphinx is auto-run, and does not seem to complain. I once had issues with "Attributes" being not understood. Can check next time.